.github/CODEOWNERS

```diff
@@ -47,7 +47,6 @@
 /x-pack/test/functional/apps/lens @elastic/kibana-vis-editors
 /x-pack/test/api_integration/apis/lens/ @elastic/kibana-vis-editors
 /test/functional/apps/visualize/ @elastic/kibana-vis-editors
-/src/plugins/unified_field_list/ @elastic/kibana-vis-editors
 /test/api_integration/apis/unified_field_list/ @elastic/kibana-vis-editors
```

Comment:
Please remove the next line too with tests

Reply:
makes sense, thx!